sample/src/App.tsx

```diff
@@ -60,8 +60,8 @@ Sentry.init({
   tracesSampleRate: 1.0,
   // Sets the `release` and `dist` on Sentry events. Make sure this matches EXACTLY with the values on your sourcemaps
   // otherwise they will not work.
-  release: packageVersion,
-  dist: `${packageVersion}.0`,
+  release: '1.2.3',
+  dist: `1.2.3.0`,
```

Comment:
should we revert this change?

Reply:
We can no longer use the version from the parent folder when the SDK is packaged and the linking to the parent folder is removed so thus why this change here